config/v0.3.0/log.go

```diff
@@ -183,7 +183,7 @@ func otlpGRPCLogExporter(ctx context.Context, otlpConfig *OTLP) (sdklog.Exporter
 		} else {
 			opts = append(opts, otlploggrpc.WithEndpoint(*otlpConfig.Endpoint))
 		}
-		if u.Scheme == "http" {
+		if u.Scheme == "http" || (u.Scheme != "https" && otlpConfig.Insecure != nil && *otlpConfig.Insecure) {
```

Comment:
Should we add tests?

Reply:
I would like to and spent some time on it already, but these options get double wrapped in structs with only private members so I was at a loss as to how to inspect the output of this method. Happy to take suggestions

Reply:
No, sorry, your reasoning makes sense.